.github/actions/setup-job-docker/action.yml

```diff
@@ -39,7 +47,25 @@ runs:
         mkdir -p cargo-installs/bin
         cp /usr/local/cargo/bin/async-bindgen-gen-dart cargo-installs/bin/async-bindgen-gen-dart
 
+    - name: Jfrog authentication
+      shell: bash
+      if: ${{ inputs.jfrog_token != 'false' }}
+      env:
+          JFROG_TOKEN: ${{ inputs.jfrog_token }}
+      run: |
+        if [ -z "${JFROG_TOKEN}" ]; then
+          echo "JFROG access token missing"
+          exit 1
+        fi
+        echo "${JFROG_TOKEN}" | dart pub token add "https://xayn.jfrog.io/artifactory/api/pub/dart.yellow.private"
+
     - name: dart deps
       shell: bash
       if: ${{ inputs.dart != 'false' }}
       run: just dart-deps
+
+    - name: bindgen dart deps
+      shell: bash
+      working-directory: async_bindgen
+      if: ${{ inputs.bindgen_dart != 'false' }}
+      run: just dart-deps
```

Comment:
This duplicates the previous step, probably not what we want.

Reply:
I removed it I also removed the copy of the async-bindgen binary (which doesn't work like that anymore, it's now handled implicitly by rust-cach)